src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -10766,6 +10766,125 @@ .maxstack 8
             CompileAndVerify(comp, expectedOutput: "System.Action`2[System.Int32[],System.Int32]");
         }
 
+        [Fact]
+        [WorkItem(63565, "https://github.com/dotnet/roslyn/issues/63565")]
+        public void SynthesizedDelegateTypes_UnscopedRefAttribute_01()
```

Comment:
>SynthesizedDelegateTypes_UnscopedRefAttribute_01 [](http://example.com/codeflow?start=20&length=48)

It feels like we should verify that the attribute is present on the delegate parameter.